schema/compose-spec.json

```diff
@@ -165,12 +165,15 @@
         "container_name": {"type": "string"},
         "cpu_count": {"type": "integer", "minimum": 0},
         "cpu_percent": {"type": "integer", "minimum": 0, "maximum": 100},
-        "cpu_shares": {"type": ["number", "string"]},
-        "cpu_quota": {"type": ["number", "string"]},
-        "cpu_period": {"type": ["number", "string"]},
-        "cpu_rt_period": {"type": ["number", "string"]},
-        "cpu_rt_runtime": {"type": ["number", "string"]},
-        "cpus": {"type": ["number", "string"]},
+        "cpu_shares": {"anyOf":[{"type":"number"},{"type":"string"}]},
+        "cpu_quota": {"anyOf":[{"type":"number"},{"type":"string"}]},
+        "cpu_period": {"anyOf":[{"type":"number"},{"type":"string"}]},
+        "cpu_rt_period": {"anyOf":[{"type":"number"},{"type":"string"}]},
+        "cpu_rt_runtime": {"anyOf":[{"type":"number"},{"type":"string"}]},
```

Comment:
I believe these should be `oneOf`. (But in practice, I don't think it makes a difference, since it's not possible for something to fulfill both types simultaneously)

Reply:
(I think this applies to all the `anyOf`s in the file, but I won't comment on each one!)